images/image_consumer2a/data_consumer2a.py

```diff
@@ -0,0 +1,56 @@
+from kafka import KafkaConsumer, KafkaProducer
+import json
+import uuid
+import pandas as pd
+import csv
+import time
+from datetime import datetime
+
+server1 = 'broker1:9093'
+server2 = 'broker2:9095'
+server3 = 'broker3:9097'
+
+topic1 = "data_gen1"
+topic2 = "data_gen2"
+
+print(f'{datetime.now()}: config..', flush=True)
+
+consumer2a = KafkaConsumer(topic2,
+                         auto_offset_reset='earliest',
+                         bootstrap_servers=[server2],
+                         api_version=(0, 10),
+                         value_deserializer = json.loads,
+                         consumer_timeout_ms=500,
```

Comment:
wenn du hier das Timeout entfernst, brauchst du den Infinite-Loop weiter unten nicht mehr.

Reply:
für consumer1 entfernt, consumer2 sollen die Nachrichten in unterschiedlichen Intervallen abholen